cob_hardware_config/components/sick_lms1xx/sick_lms1xx.yaml

```diff
@@ -0,0 +1,3 @@
+angle_resolution: 0.25 #deg
+scan_frequency: 25 #hz
+set_config: false
```

Comment:
Here, the same holds as for the s300 below. Those parameter might need to be changed depending on the instance of the sensor, not the component of "lms1xx" in general.

You can configure the scanner to have a resolution of `0.25` or `0.5` degrees (probably even some more). You could even have one of each on one single robot.

Do you see my point?